homeassistant/components/renson/renson_sensor.py

```diff
@@ -0,0 +1,51 @@
+"""Sensor class of the Renson ventilation system."""
+from renson_endura_delta.general_enum import DataType
+import renson_endura_delta.renson as renson
+
+from homeassistant.components.renson.renson_descriptions import (
+    RensonSensorEntityDescription,
+)
+from homeassistant.components.sensor import SensorEntity
+
+
+class RensonSensor(SensorEntity):
```

Comment:
For the first PR, we need to keep the integration as minimal as possible.

Please remove the `sensor` platform from this integration for now and start with just `binary_sensor` (or the other way around).

We can add more platforms in the next PRs after this one.

See also: <https://developers.home-assistant.io/docs/creating_component_code_review?_highlight=small#5-make-your-pull-request-as-small-as-possible>